Source/Scene/PostProcessStageComposite.js

```diff
@@ -245,26 +245,6 @@ define([
         return true;
     };
 
-    /**
-     * Whether or not this post process stage is supported.
-     * <p>
-     * A post process stage is not supported when it requires a depth texture and the WEBGL_depth_texture extension is not
-     * supported.
-     * </p>
-     *
-     * @param {Scene} scene The scene.
-     * @return {Boolean} Whether this post process stage is supported.
-     *
-     * @see {Context#depthTexture}
-     * @see {@link http://www.khronos.org/registry/webgl/extensions/WEBGL_depth_texture/|WEBGL_depth_texture}
-     */
-    PostProcessStageComposite.prototype.isSupported = function(scene) {
```

Comment:
Should `PostProcessStage.prototype.isSupported` also be removed?

Reply:
Yes, removed.